client/app/assets/less/inc/base.less

```diff
@@ -100,6 +100,10 @@ strong {
   cursor: pointer;
 }
 
+.clickable:disabled {
```

Comment:
Not sure there's an action to take, yet, but I have mixed feelings about this. Anything can take the classname `clickable`, but not anything can be `disabled`.